src/Fantomas/AstTransformer.fs

```diff
@@ -562,7 +562,7 @@ module private Ast =
              Properties = p []
              FsAstNode = synExpr
              Childs = [yield visitSynExpr expr]}
-        | SynExpr.LetOrUseBang(_,isUse,isFromSource,pat,rhsExpr,body,range) ->
+        | SynExpr.LetOrUseBang(_,isUse,isFromSource,pat,rhsExpr,andBangs,body,range) ->
```

Comment:
andBangs are a list of tuples, where the 4th and 5th items are the pattern and body expr, respectively.  so we iterate them in this order here